test/support/user.ex

```diff
@@ -34,9 +35,13 @@ defmodule Crudry.User do
 
   defp base_changeset(user, attrs) do
     user
-    |> cast(attrs, [:username, :age, :company_id])
+    |> cast(attrs, [:username, :age, :password, :company_id])
     |> validate_required([:username])
     |> validate_length(:username, min: 2)
     |> validate_number(:age, greater_than: 0)
+    |> validate_length(:password, min: 8)
+    |> validate_format(:password, ~r/[0-9]+/, message: "Password must contain a number")
```

Comment:
Ah, it's just the message that is wrong. I think removing the `"Password "` here makes more sense, since the changeset error will already be something like `%{password: "must contain a number"}`